api/toc.yml

```diff
@@ -1,6 +1,6 @@
-- name: Reference
+- name: API reference
   items:
-  - name: Q# libraries
+  - name: Q# libraries (API)
```

Comment:
It feels odd to me to repeat "API" here, especially when the ".NET libraries" item isn't qualified in the same way?

Reply:
Yeah that's what I was unsure of too. I think if we keep the overarching section titled "API reference" then we're good to remove this